assets/js/modules/adsense/datastore/constants.js

```diff
@@ -26,6 +26,6 @@ export const SITE_STATE_REQUIRES_REVIEW = 'REQUIRES_REVIEW';
 export const SITE_STATE_GETTING_READY = 'GETTING_READY';
 export const SITE_STATE_READY = 'READY';
 
-export const STATE_NEEDS_ATTENTION = 'NEEDS_ATTENTION';
-export const STATE_REQUIRES_REVIEW = 'REQUIRES_REVIEW';
-export const STATE_GETTING_READY = 'GETTING_READY';
+export const API_STATE_NEEDS_ATTENTION = 'NEEDS_ATTENTION';
+export const API_STATE_REQUIRES_REVIEW = 'REQUIRES_REVIEW';
+export const API_STATE_GETTING_READY = 'GETTING_READY';
```

Comment:
Can you please remove the above `SITE_STATE_*` constants and only keep the `API_STATE_*` ones? We can use the latter for any states coming from the API, no matter if it is for accounts, clients, or sites, since the possible values for all 3 types of entities are the same. Let's add a `API_STATE_READY` to complete the new list.